package.json

```diff
@@ -109,7 +109,7 @@
     "react-dom": "^18.3.1",
     "react-joyride": "^2.7.2",
     "rimraf": "^3.0.2",
-    "storybook": "^9.0.0 || ^9.0.0-alpha.4",
+    "storybook": "0.0.0-pr-30875-sha-5d0152e4",
```

Comment:
I will revert these versions back to 9.0.0 alphas before merging this PR, but we're waiting for a new alpha to release with the new TP API usage.